src/Elemental.js

```diff
@@ -1,3 +1,4 @@
+require('babel/register');
```

Comment:
Understood.  Half-baked code trying to work out the details of fixing window dependencies.